### ISSUE TITLE
Today's Articles not populating

### ISSUE THREAD

**Issue by petertomaszewski** [ViennaRSS/vienna-rss#1804]
**Describe the bug**
Before I updated to the latest version, if I clicked on Today’s Articles the articles dated today would populate. Same for Unread Articles and Marked Articles. Now these are not working. I’m wondering if it’s just my Mac? This was not the case before I updated Vienna this week. Thanks.

**To Reproduce**
When I open Vienna no articles populate

**Screenshots**
![Greenshot 2024-08-29 19 37 23](https://github.com/user-attachments/assets/b7ec798f-1bf9-4b78-bea5-131f5d445f63)

**Please complete the following information:**
 - Vienna version [e3.9.2]
 - OS version: [macOS 14.6.1]

**Additional information:**
In Vienna before 3.9.2 this was not an issue. this just started after I upgraded versions.

**Comment by barijaona** [comment]
So you previously used 3.9.1 ?
Have you checked that the articles appear when you select individual feeds ?

**Comment by petertomaszewski** [comment]
Yes I used 3.9.1 and the articles do appear in the individual feeds.


--
Peter Tomaszewski

SR3 Cybersecurity
simple.secure.sustainable.
reliable.redundant.resilient.

On Aug 29, 2024 at 22:44 -0500, Barijaona Ramaholimihaso ***@***.***>, wrote:

So you previously used 3.9.1 ?
Have you checked that the articles appear when you select individual feeds ?

**Comment by barijaona** [comment]
I don't see any reason why this would have been caused by the upgrade from 3.9.1 to 3.9.2
I suggest you have a look at the smart folder definitions by double-clicking on them : verify if anything seems strange to you and re-validate.

**Comment by mr-sk** [comment]
I have the same issue after upgrading to latest. 

Version: 3.9.2
OSX: Darwin ABC 23.6.0 Darwin Kernel Version 23.6.0: Mon Jul 29 21:14:46 PDT 2024; root:xnu-10063.141.2~1/RELEASE_ARM64_T6031 arm64
Mac: 14.6.1

The "smart" folders (Today, Unread, Marked) folders are all empty. This is a new bug, always worked in the past. No changes to the settings.

**Comment by barijaona** [comment]
Have you tried creating a new smart folder ? Did it worked ?

**Comment by petertomaszewski** [comment]
No it did not work.


--
Peter Tomaszewski

SR3 Cybersecurity
simple.secure.sustainable.
reliable.redundant.resilient.

On Oct 9, 2024 at 05:43 -0500, Barijaona Ramaholimihaso ***@***.***>, wrote:

Have you tried creating a new smart folder ? Did it worked ?

**Comment by barijaona** [comment]
Which localization of Vienna is used in your configuration ? English or another language ?

May I have a screenshot of the Today smart folder ? Mine (in French) looks like this :

<img width="658" alt="Capture d’écran 2024-10-10 à 09 35 17" src="https://github.com/user-attachments/assets/434fb51b-65cd-4359-8bfd-ed9cd180d852">

**Comment by petertomaszewski** [comment]
My primary language is English. This is the screenshot of the Today’s Articles folders. Seems the last upgrade removed some items?

[cid:285251CABB9B47ED88E934BAB43CBDB4]


--
Peter Tomaszewski

SR3 Cybersecurity
simple.secure.sustainable.
reliable.redundant.resilient.

On Oct 10, 2024 at 1:36 AM -0500, Barijaona Ramaholimihaso ***@***.***>, wrote:

Which localization of Vienna is used in your configuration ? English or another language ?

May I have a screenshot of the Today smart folder ? Mine (in French) looks like this :

Capture.d.ecran.2024-10-10.a.09.35.17.png (view on web)<https://github.com/user-attachments/assets/434fb51b-65cd-4359-8bfd-ed9cd180d852>

**Comment by barijaona** [comment]
Unfortunately, as you replied by e-mail, the screenshot you attached did not reach me.

Are you able to repair things to look like this ?
<img width="661" alt="Capture d’écran 2024-10-11 à 13 00 26" src="https://github.com/user-attachments/assets/5f0f5669-9720-4f18-9784-c93e87ad72e6">

**Comment by petertomaszewski** [comment]
My folder does not have the filters yours does. My folder is empty. I'm on a Mac, how do I add the filters to my folder. Are there instructions?
<img width="665" alt="Greenshot 2024-10-11 10 57 59" src="https://github.com/user-attachments/assets/2d594e69-3f88-4109-86da-68aa28f0169c">

**Comment by petertomaszewski** [comment]
I had to add 3 new smart folders and create the rules and now it works like before. I suspect the rules got taken out of the predefined smart folders in the upgrade. You should have a knowledge base article on how to create new smart folders and add rules.

**Comment by TAKeanice** [comment]
Hi @petertomaszewski , I agree that an explanation on how smart folders work could be beneficial to for help pages. If you know how they work now, you could contribute to Vienna by writing the article and send it to one of us for adaptation and inclusion into the next version. From then on, your favorite RSS reader software would always contain a piece of your work!

**Comment by mpechner** [comment]
OSX, nn Version 3.9.5 :7784c797: (8414)
The default smart folder are not working anymore "Today's articles", "unread articles" and "Marked Articles"

Even checked the filterbar to make sure the settings are correct.

Not sure how writing an explanation on smart folder will help a when the default smart folders do not work anymore.
